inspire/base/templates/header.html

```diff
@@ -434,19 +421,36 @@ <h5 class="text-left"><ins><a class="cursor-pointer popover-link">External resou
   </nav>
 </section>
 
-{% if request.path  == "/search" %}
-  <div id="collections">
-    <div class="container">
-      {% set collection_name = request.args.get('cc') %}
-        <span id="{{collection_name | collection_select_current("Literature") }}">Literature</span>
-        <span id="{{collection_name | collection_select_current("Authors") }}">Authors</span>
-        <span id="{{collection_name | collection_select_current("Conferences") }}">Conferences</span>
-        <span id="{{collection_name | collection_select_current("Jobs") }}">Jobs</span>
-    </div>
-  </div>
-  <div class="collection-header" id="{{collection_name | lower }}-collection">
+
+{#
+## If not home path, show collections bar
+#}
+{% if request.path != '/' %}
```

Comment:
You can leave it hidden for the moment, I think it will be better to have the Literature collection as our front page.